etc/sql/sql.sql

```diff
@@ -158,6 +158,10 @@ CREATE TABLE files (
     owner_id STRING,
     storage_type STRING,
     content_type STRING,
+    dimensions OBJECT(STRICT) AS (
```

Comment:
This is a generic file table, so why not using a generic name for additional data such as `info` and do not define the structure of the object `OBJECT(IGNORED)`.
This will allow to use this for other data too (e.g. file size).